doc/api/test.md

```diff
@@ -1360,16 +1360,16 @@ changes:
     If `false`, it would only run one test file at a time.
     **Default:** `false`.
   * `cwd`: {string} Specifies the current working directory to be used by the test runner.
-    Serves as the base path for resolving files according to the [test runner execution model][].
+    Serves as the base path for resolving files the same as the [`node --test`](#running-tests-from-the-command-line) command line.
```

Comment:
Can you add it as reference at the end of the file?
```suggestion
    Serves as the base path for resolving files the same as the [`node --test`][] command line.
```

Reply:
done.